core/constants_test.py

```diff
@@ -64,6 +64,20 @@ def test_loading_file_in_package_returns_the_content(self) -> None:
                 'File data'
             )
 
+        with utils.open_file(
+            os.path.join(
+                'assets', 'images', 'avatar', 'user_blue_150px.png'),
+            'rb',
+            encoding=None
+        ) as f:
+            raw_image_png = f.read()
+        default_image_path = os.path.join(
+            'images', 'avatar', 'user_blue_150px.png')
+        self.assertEqual(
+            constants.get_package_file_contents(
+                'assets', default_image_path, mode='rb'), raw_image_png
+        )
```

Comment:
Hmm, can you also test the same in the case when the `io.read` is not working?

Reply:
Added in the below mentioned test.